class-affiliates-ninja-forms.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * affiliates-ninja-forms.php
+ *
+ * Copyright (c) 2012-2015 "kento" Karim Rahimpur www.itthinx.com
+ *
+ * This code is released under the GNU General Public License.
+ * See COPYRIGHT.txt and LICENSE.txt.
+ *
+ * This code is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * This header and all notices must be kept intact.
+ *
+ * @author Karim Rahimpur
+ * @package affiliates-ninja-forms
+ * @since affiliates-ninja-forms 1.0.0
+ *
+ * Plugin Name: Affiliates Ninja Forms Integration
```

Comment:
@eggemplo you must remove the plugin tags, these must only be defined once in the main plugin file